client/tests/sdk/test_dataset.py

```diff
@@ -1511,9 +1508,15 @@ def test_row(self) -> None:
 
 
 class TestRowWriter(BaseTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.workdir = Path(self.local_storage) / ".dataset_tmp"
```

Comment:
Is it better to use `os.chdir`? 
And then there is no need to specify the `workdir` param in each test.

Reply:
updated